invenio_rdm_records/records/jsonschemas/records/record-v6.0.0.json

```diff
@@ -352,21 +352,21 @@
     },
     "tombstone": {
       "type": "object",
-      "description": "Tombstone for the record.",
+      "description": "Tombstone information for the record.",
       "additionalProperties": false,
       "properties": {
-        "reason": {
+        "removal_reason": {
           "type": "string",
           "description": "Reason for removal."
         },
-        "category": {
+        "note": {
           "type": "string",
-          "description": "Category for the removal."
+          "description": "Public note about the removal."
         },
         "removed_by": {
```

Comment:
Question: Probably this should be using the entity resolver?

Reply:
I had that thought too, but wouldn't that mean that we would be displaying something like `removed by Max Moser` rather than `removed by staff`?
Unless we want to go into some sort of heuristic magic with checking the roles of resolved entities at display time (which is bound to misfire from time to time)?

TBH, I wouldn't mind actually having the `removed by Max Moser` there, but that would differ from the RFC.

Reply:
Yes, I was thinking that we could probably simply handle this during rendering.....ie..e we store the specific person, but rendering we can decide what todo.

Reply:
perfect, we should be able to achieve this sort of behavior with the current implementation!